app/styleguides/_default/components/tweet.css

```diff
@@ -0,0 +1,48 @@
+.clay-tweet {
+    display: block;
+    margin: 0 0 24px;
+    width: 100%;
+
+    & blockquote.twitter-tweet {
+      border-color: #eee #ddd #bbb;
+      border-radius: 5px;
+      border-style: solid;
+      border-width: 1px;
+      box-shadow: 0 1px 3px rgba(0, 0, 0, .15);
+      display: inline-block;
+      font-family: 'Helvetica Neue', Roboto, 'Segoe UI', Calibri, sans-serif;
+      font-size: 12px;
+      font-weight: 700;
+      line-height: 16px;
+      margin: 10px 5px;
+      max-width: 468px;
+      padding: 0 16px 16px;
+    }
+
+    & blockquote.twitter-tweet p {
+      font-size: 16px;
+      font-weight: 400;
+      line-height: 20px;
+    }
+
+    & blockquote.twitter-tweet a {
+      color: #1782a9;
+      font-weight: 400;
+      outline: 0;
+      text-decoration: none;
+    }
+
+    & blockquote.twitter-tweet a:focus,
+    & blockquote.twitter-tweet a:hover {
+      text-decoration: underline;
+    }
+
+    @media screen and (min-width: 768px) {
+      margin: 0 0 20px;
```

Comment:
Can you confirm that this actually works? Seems odd to see a property with no selector